app/models/spree_amazon/configuration.rb

```diff
@@ -0,0 +1,10 @@
+class SpreeAmazon::Configuration < Spree::Preferences::Configuration
+  preference :client_id, :string
+  preference :merchant_id, :string
+  preference :aws_access_key_id, :string
+  preference :aws_secret_access_key, :string
+
+  def use_static_preferences!
+    raise "SpreeAmazon::Configuration cannot use static preferences"
+  end
```

Comment:
Ah so whenever someone calls `#use_static_preferences!` the preferences get reset to their default values and they will be stored in memory from that call onwards, which doesn't work for us if we want to keep the Admin UI, we need to persist them in the database.

You can take a look at the code [here](https://github.com/solidusio/solidus/blob/master/core/app/models/spree/preferences/configuration.rb#L49-L51)